packages/components/src/divider/src/Divider.tsx

```diff
@@ -0,0 +1,61 @@
+import { useStyledSystem, type StyledComponentProps } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { type CSSProperties, forwardRef, type ForwardedRef } from "react";
+import { Separator as RACSeparator, useContextProps, type SeparatorProps as RACSeparatorProps } from "react-aria-components";
+
+import { cssModule } from "../../utils/index.ts";
+
+import { DividerContext } from "./DividerContext.ts";
+
+import styles from "./Divider.module.css";
+
+export const GlobalDividerCssSelector = "hop-Divider";
+
+export interface DividerProps extends StyledComponentProps<RACSeparatorProps> {}
+
+function Divider(props: DividerProps, ref: ForwardedRef<HTMLElement>) {
+    [props, ref] = useContextProps(props, ref, DividerContext);
+    const { stylingProps, ...ownProps } = useStyledSystem(props);
+    const {
+        className,
+        style,
+        orientation,
+        ...otherProps
+    } = ownProps;
+
+    const classNames = clsx(
+        GlobalDividerCssSelector,
+        cssModule(
+            styles,
+            "hop-Divider",
+            orientation
+        ),
+        stylingProps.className,
+        className
+    );
+
+    const mergedStyles: CSSProperties = {
+        ...stylingProps.style,
+        ...style
+    };
+
+    return (
+        <RACSeparator
+            {...otherProps}
+            ref={ref}
+            className={classNames}
+            style={mergedStyles}
+            orientation={orientation}
+        />
+    );
+}
+
+/**
+ * The Divider separates the distinguishes sections of content or groups.
+ *
+ * [View Documentation](TODO)
+ */
+const _Divider = forwardRef<HTMLElement, DividerProps>(Divider);
```

Comment:
Wouldn't it be better `HTMLHRElement`?

Reply:
I have it as HTMLElement because it's not always an HR. It can be a vertical divider as well. By default, RAC changes it to a span for this type. The user also has the ability to change the html element if I remember right.

Reply:
I hadn't noticed the vertical separation. The solution you adopted is the right one.